sdk/keyvault/Azure.Security.KeyVault.Certificates/src/PemReader.cs

```diff
@@ -29,6 +29,7 @@ internal static partial class PemReader
 
             if (s_ecCopyWithPrivateKeyMethod is null)
             {
+                // CopyWithPrivateKey was added in .NET Framework 4.7.2. We must otherwise throw without it.
```

Comment:
Are the keyvault changes needed as part of this change?

Reply:
Oops. No. I'll rebase on main since they went in yesterday.